### DISCUSSION TITLE
How to import nextPYP generated STAR file to RELION

### DISCUSSION THREAD

**Discussion by JuhaHuiskonen** [nextpyp/.github#82] [Q&A]
Hi, I have exported the project to RELION.  I have a `particles.star` and `tomograms.star` files.  I have imported these to RELION 5.0 using `Tomo Import` and options `Import Coordinates` and `Other` (Tomogram set). These imports seem to work. However, when I try to reconstruct the tomograms, I get garbage out. Also running Reconstruct particle does not give a correct average. Previously when I reconstructed tomograms in RELION from scratch, they were oriented differently. So I suspect I will need to use `Flip YZ` and possibly `Flip Z` options. These are available for `Import tomograms` option, but this option does not work with the `tomograms.star` file (RELION crashes).

**Comment by yezhou867** [comment]
Hi, we were testing the export relion format using RELION 4.0.  Haven't tested the most recent RELION5.0.  We may need to figure out what has been changed for the new version.

**Comment by JuhaHuiskonen** [comment]
Thanks for the speedy response.  RELION 4 `Tomo import` and its `Import tomograms` function expect IMOD directories to be present.  These are not included when exporting from nextPYP.  So how exactly should the nextPYP generated STAR files (tomograms and particles) imported to RELION 4? Thanks.

**Reply by yezhou867** [reply]
I think you can ignore the imod folder requirement, and directly import paticles.star.  Then make pseudo-subtomos and run the reconstruction.  Tomograms.star will be used in the make pseudo-subtomograms step. 
I'm testing relion5, I think you can also use relion 5 `Tomo import`  `Coordinates` tab only with both particle.star and tomograms.star and leave other importing methods tabs empty (only give it general info).

**Reply by JuhaHuiskonen** [reply]
Are you able to reconstruct the tomograms back in RELION 5 using the imported tomograms.star?

**Reply by yezhou867** [reply]
I did get a reconstruction with Class3d using `skip_align` option.  However, the quality is different since the weighting effect is not there anymore. 
 the input `--ios PseudoSubtomo/job002/optimisation_set.star` looks like this
```
data_

_rlnTomoParticlesFile            PseudoSubtomo/job002/particles.star
_rlnTomoTomogramsFile            tomo-coarse-refinement-f00ZkzM7w9Fl8eNA_tomograms.star
```

**Reply by JuhaHuiskonen** [reply]
I meant actually tomograms, not subtomograms.  So were you able to reconstruct tomograms in RELION using the imported `tomograms.star`? I am asking as I would like to double check that the coordinates are picked correctly.

**Comment by yezhou867** [comment]
<img width="776" alt="Screenshot 2024-03-15 at 12 07 09 PM" src="https://github.com/nextpyp/.github/assets/139471026/b87e0661-0782-4105-aaed-8c699ebf4c08">
Sorry, it takes longer for me to get a chance to do a test with "tomograms.star" for reconstruction in relion5.  It seems to be able to get a reconstruction, while the filtering effect looks different. I have to look at more cases to learn about RELION5 to see what is the important factor to work properly.
